models.go

```diff
@@ -4,25 +4,14 @@ import (
     "time"
 )
 
-type Device struct {
-    ID int
-    CustomerID int `gorm:"not null"`
-    DeviceName string `gorm:"not null unique"`
-    LastHeartbeat time.Time
-    IsActive bool `gorm:"not null"`
-    PartyName string `gorm:"size:50"`
-    PartySize int
-    WaitTime int
-}
-
 type Restaurant struct {
-    restaurantID int
+    id int
     name string `gorm:"size:99; not null"`
     dateCreated time.Time `gorm:"not null"`
 }
 
 type ActiveParty struct {
-    activePartyID int
+    id int
     restaurant Restaurant
```

Comment:
are we sure these foreign keys will work now that you changed restaurantID to just id which they all have